tests/unit/replicator_tests.py

```diff
@@ -116,6 +118,7 @@ def test_replication_with_generated_id(self):
         clone = Replicator(self.client)
         clone.create_replication(self.db, self.target_db)
 
+    @flaky(max_runs=3)
```

Comment:
I'm unsure. The test looks fine to me in that it creates a valid rep document. The issue seems to be with the service not reliably starting the replication. Maybe a latent bug?